pkg/cmd/controller/csr.go

```diff
@@ -0,0 +1,54 @@
+package controller
+
+import (
+	"context"
+
+	"github.com/openshift/cluster-policy-controller/pkg/operator"
+	"github.com/openshift/library-go/pkg/operator/csr"
+	"github.com/openshift/library-go/pkg/operator/events"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+)
+
+const (
+	controllerName = "csr-approver-controller"
+	// Hardcoded for the Monitoring Operator:
+	// https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/0000_50_cluster-monitoring-operator_03-role-binding.yaml#L25-L27
```

Comment:
reference the commit id as master will change https://github.com/openshift/cluster-monitoring-operator/blob/e7a2b2e20d19ba7a9271df59e06b1fd32b8bdfe4/manifests/0000_50_cluster-monitoring-operator_03-role-binding.yaml#L25-L27

Reply:
No, I'm not very keen on doing this. If this file gets moved, the ID as SHA-1 will point to a still-existing file. This will give proper pointers where to look next.